### ISSUE TITLE
Migration fail? No Such column: Series.TRAKT_ID

### ISSUE THREAD

**Issue by SchizoDuckie** [SchizoDuckie/DuckieTV#732]
Upon opening DuckieTV today I was presented with this:

![image](https://cloud.githubusercontent.com/assets/111710/16743224/97ef1114-47ab-11e6-8374-e28d4a737e4c.png)

strange, because the last time i pulled was somewhere last week afairc.

Did I miss a migration? ( still have to investigate, will help out with #675 )

``````
SELECT Series.ID_Serie, 
    Series.name, 
    Series.banner, 
    Series.overview, 
    Series.TVDB_ID, 
    Series.IMDB_ID, 
    Series.TVRage_ID, 
    Series.actors, 
    Series.airs_dayofweek, 
    Series.airs_time, 
    Series.timezone, 
    Series.contentrating, 
    Series.firstaired, 
    Series.genre, 
    Series.country, 
    Series.language, 
    Series.network, 
    Series.rating, 
    Series.ratingcount, 
    Series.runtime, 
    Series.status, 
    Series.added, 
    Series.addedby, 
    Series.fanart, 
    Series.poster, 
    Series.lastupdated, 
    Series.lastfetched, 
    Series.nextupdate, 
    Series.displaycalendar, 
    Series.autoDownload, 
    Series.customSearchString, 
    Series.watched, 
    Series.notWatchedCount, 
    Series.ignoreGlobalQuality, 
    Series.ignoreGlobalIncludes, 
    Series.ignoreGlobalExcludes, 
    Series.searchProvider, 
    Series.ignoreHideSpecials, 
    Series.customSearchSizeMin, 
    Series.customSearchSizeMax, 
    Series.TRAKT_ID
 FROM 
    Series
  WHERE name is not NULL   LIMIT 0,1000``` 
``````

**Comment by SchizoDuckie** [comment]
Interesting:

`database.version.Episodes: "13"`

**Comment by SchizoDuckie** [comment]
Hmm okay, so after juggling versions i got the last migration to execute again.

Train of thought: I can see if i can patch CreateReadUpdateDelete.js so that it verifies the create statement that shows when fetching from SQLITE_MASTER where name = $table after a migration has executed, to verify if it has executed completely and correctly.

**Comment by garfield69** [comment]
ran a migration on my win7 sandbox from DuckieTV standalone 1.1.3 (nwjs 0.12.3) to 1.1.4 (nwjs 0.14.7 dev-build) and all steps were completed successfully \o/
So was this an issue with your source management and can we close the ticket, or is there some undetermined/intermittent issue that still need investigating?

**Comment by SchizoDuckie** [comment]
I think the testcase is more likely a situation where the DuckieTV tab was closed before all migrations were finished. Nearly impossible to test, but i think the latest updates helps mitigate that mostly

**Comment by garfield69** [comment]
@SchizoDuckie  This [Reddit](https://www.reddit.com/r/duckietv/comments/4w36cr/blank_calendar_after_update/) documents a user that upgraded from DuckieTV 1.1.3 standalone to the latest nightly.
He ended up with an empty calendar, and unable to view episodes for his shows.     
After examining his database at https://mega.nz/#!yE8GSDQb!qOlOlI30wUa5qS2z2AiODP2dLn5gFS3AwAlrSVaMH-A I determined that the migration of the Episodes table failed between versions 11 to 12 to 13.    
The fix was to get the user to issue the following at the console prompt and restart DuckieTV.    
```
localStorage.setItem('database.version.Episodes', 11);CRUD.executeQuery('drop table Seasons_bak');
```
I added the Seasons_bak drop because I found that table even though it appears all the Seasons table data was populated, and did not know if having it there would defeat a subsequent Seasons table migration in the future.
